server/container_create.go

```diff
@@ -648,7 +648,12 @@ func (s *Server) setupSeccomp(specgen *generate.Generator, profile string) error
 		return nil
 	}
 	if profile == seccompRuntimeDefault || profile == seccompDockerDefault {
```

Comment:
Can we add some comments starting from here to this method? The code is not entirely obvious.

Reply:
Did this to make the intention more clear.